src/main/ipc.js

```
@@ -492,43 +492,131 @@ function setupIpcHandlers(windows) {
+        console.log(`Performing manual sync action: ${action}`);
+        const result = await cloudSyncManager.manualSync(action);
+
+        // 동기화 결과가 성공이면 UI 업데이트 메시지 전송
+        if (result.success) {
+          // 현재 설정 데이터 수집
+          const configData = {
+            pages: config.get('pages'),
+            appearance: config.get('appearance'),
+            advanced: config.get('advanced'),
+            subscription: config.get('subscription')
+          };
```

ESLint

[error] Missing trailing comma.

(comma-dangle)